ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -20,7 +20,7 @@ let prototyped_of_field = function
   | "host", "last_software_update" ->
       Some "22.20.0"
   | "pool", "migration_compression" ->
-      Some "22.32.0-next"
+      Some "22.33.0"
```

Comment:
This will conflict with @robhoes 's change, rebase the PR after we merge his.

Reply:
Already merged: https://github.com/xapi-project/xen-api/commit/cde2822208d1383597ff6ffa4394ca5b6b39637a